lib/super_digit.rb

```diff
@@ -1,15 +1,31 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity O(n)
+# Space Complexity Constant
 def super_digit(n)
```

Comment:
The time complexity is O(d^2) where d is the number of digits.  because the sum will take d times and it's in a recursive method which runs d times.